example.js

```diff
@@ -0,0 +1,353 @@
+var mutate = require('xtend/mutable')
+var expose = require('choo-expose')
```

Comment:
Derp, yeah I should publish it, it's still just on my machine - 10 lines tho but yeah lol